attestation/product/product.go

```diff
@@ -174,16 +184,28 @@ func (a *Attestor) Attest(ctx *attestation.AttestationContext) error {
 }
 
 func (a *Attestor) MarshalJSON() ([]byte, error) {
-	return json.Marshal(a.products)
+	output := attestorJson{
+		Products: a.products,
+		Configuration: attestorConfiguration{
```

Comment:
If we make Configuration optional, we can construct the configuration prior to this and check if both globs are `""`.

Reply:
Yes that possible. I'll check on that.

Reply:
Fixed this.